### DISCUSSION TITLE
Welcome to three-mesh-ui Discussions!

### DISCUSSION THREAD

**Discussion by felixmariotto** [felixmariotto/three-mesh-ui#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phdcoder** [comment]
Hi ! Alessandro here!
Frist of all, thank you for your hard work. I am using your lib in my phd degree project(after trying a lot of alternatives).  Great work ! I am not a good js dev to help but sure i can be a tester !!

**Reply by felixmariotto** [reply]
Thanks for the supportive words ! What is your phd project exactly ?

**Reply by phdcoder** [reply]
Its about objective video quality metrics.
